liblouis/compileTranslationTable.c

```diff
@@ -2160,10 +2160,9 @@ compileBrailleIndicator(FileInfo *nested, const char *ermsg,
 	CharsString cells;
 	if (getToken(nested, &token, ermsg))
 		if (parseDots(nested, &cells, &token))
-			if (!addRule(nested, opcode, NULL, &cells, 0, 0, newRuleOffset, NULL, noback,
-						nofor, table))
-				return 0;
-	return 1;
+			return addRule(nested, opcode, NULL, &cells, 0, 0, newRuleOffset, NULL,
+					noback, nofor, table);
+	return 0;
```

Comment:
Here you change the functionality. Before it returned 1 if `!getToken || !parseDots || addRule`. Ah, I see, you're saying this is (was) a bug. Yes that could be so.

Reply:
Yes I know, sometimes I do change the behavior. I corrected the return values in those cases where I think they were wrong.